### DISCUSSION TITLE
下好的v3.6.7包，要求强制升级至v3.6.7

### DISCUSSION THREAD

**Discussion by sugar-hit** [alibaba/arthas#2374] [Q&A]
下载 release 下的 3.6.7 版的 arthas，
scp 拷到了测试环境，结果强制检查更新、因为测试环境无法联网导致下载文件缺失。。
我就是想确认下，是不是不支持纯离线使用了呀。 有点emo。

**Comment by hengyunabc** [comment]
贴出具体的信息，复制时要把整个文件夹复制过去。只复制arthas-boot是不行的。或者去下载全量包。
